src/api/app/models/owner_search/base.rb

```diff
@@ -18,6 +18,7 @@ def devel_disabled?(project = nil)
     def initialize(params = {})
       self.params = params
       self.attribute = AttribType.find_by_name!(params[:attribute] || 'OBS:OwnerRootProject')
+      raise AttributeNotSetError unless attribute
```

Comment:
`AttribType.find_by_name!` will already raise a `UnknownAttributeTypeError` if it can't find an attribute by that name. 

```
[2] pry(main)> AttribType.find_by_name!('hans:franz')
  AttribType Load (0.4ms)  SELECT `attrib_types`.* FROM `attrib_types` INNER JOIN `attrib_namespaces` ON `attrib_namespaces`.`id` = `attrib_types`.`attrib_namespace_id` WHERE (attrib_namespaces.name = 'hans' and attrib_types.name = 'franz') LIMIT 1
AttribType::UnknownAttributeTypeError: Attribute Type hans:franz does not exist (AttribType::UnknownAttributeTypeError)
from /obs/src/api/app/models/attrib_type.rb:47:in 'AttribType.find_by_namespace_and_name'
```